docs/README.md

```diff
@@ -39,14 +39,18 @@ To build the documentation locally:
 
 1. Clone the ExecuTorch repo to your machine.
 
-1. If you don't have it already, start a conda environment:
+   ```bash
+   git clone -b release/0.6 https://github.com/pytorch/executorch.git --depth 1 --recurse-submodules --shallow-submodules && cd executorch
```

Comment:
Can we stop doing this 

```
--depth 1 --recurse-submodules --shallow-submodules && cd executorch
```

./install_executorch.sh script already does git submodule sync automatically underneath the hood now.